examples/amberwebauthqmlexample/amberwebauthqmlexample.qml

```diff
@@ -54,7 +59,7 @@ ApplicationWindow {
                     onReceivedAccessToken: {
                         console.log("Got access token: " + oauthToken + "; " + oauthTokenSecret)
                         xhr = authenticatedRequest("GET", "https://api.twitter.com/1.1/account/verify_credentials.json",
-                                                   oauth1.generateAuthorizationHeader(
+                                                   oauth.generateAuthorizationHeader(
```

Comment:
should this be twitterOAuth.generateAuthorizationHeader() ?

Reply:
Hmm... I don't think so, as this is accessing the `oauth` property of `twitterOAuth`; i.e. [this guy here](https://github.com/sailfishos/amber-web-authorization/blob/5b04ef9776644cdecbcafddbbbba162077946234/import/OAuth10a.qml#L53), to eventually hit [this function](https://github.com/sailfishos/amber-web-authorization/blob/5b04ef9776644cdecbcafddbbbba162077946234/lib/oauth1.h#L62).

I may be misunderstanding, but `OAuth10a` doesn't inherit the function from `OAuth1`, although maybe it would make sense for the API to add an alias?

Reply:
Ah, I see.  No, it's probably fine as-is.  I just forgot that the twitterOAuth thing was a compound wrapper, consisting of both the oauth and listener.  Please ignore my comment :-)